tornadoes/tests/unit/test_tornadoes.py

```diff
@@ -170,26 +172,24 @@ def test_count_specific_query(self):
         response = self._verify_status_code_and_return_response()
         self.assertEqual(response["count"], 1)
 
-    def test_count_specific_query_with_parameters(self):
+    def test_count_specific_query_with_illegal_parameters(self):
         source = {"query": {"term": {"_id": "171171"}}}
         source = self._set_count_query(source)
         parameters = {'refresh': True}
         self.es_connection.count(callback=self.stop, source=source, parameters=parameters)
         response = self.wait()
-        response_dict = self._verify_response_and_returns_dict(response)
-        self.assertEqual(response_dict["count"], 1)
-        self.assertTrue(response.request.url.endswith('_count?refresh=True'))
+        if self.version < 5:
```

Comment:
I don't know where you are getting the version parameter, 
but it seems that you should know in the test which version it is and assert according to it.
Maybe separate to 2 unit tests with two different versions